### PR TITLE
Add citation text

### PR DESCRIPTION
**Pull request by stefaniebutland**
DONE: We may want to edit the citation text manually in Zenodo first, then modify this text accordingly

### CODE REVIEW

File: index.qmd [hunk 1]
```diff
@@ -33,6 +33,14 @@ This Earthdata Cloud Cookbook is being developed as a cross-DAAC collaboration b
 
 ## Citation
 
+You can cite The NASA Earthdata Cloud Cookbook through the project’s Zenodo archive using DOI: [10.5281/zenodo.7786710](https://doi.org/10.5281/zenodo.7786710). This DOI represents all versions, and will always resolve to the latest one.
+
+The citation will look something like:
+
+> Julia Stewart Lowndes, Luis López, Erin Robinson, ... Zachary Moon. (2023). NASA-Openscapes/earthdata-cloud-cookbook: NASA EarthData Cloud Cookbook v2023.03.0 (v2023.03.0). Zenodo. https://doi.org/10.5281/zenodo.7786711
```
Comment (stefaniebutland) [inline index.qmd:40]:
```suggestion
> Andy Barrett, Chris Battisto, Brandon Bottomley, Aaron Friesz, Alexis Hunzinger, Mahsa Jami, Alex Lewandowski, Bri Lind, Luis López, Jack McNelis, Cassie Nickles, Catalina Oaida Taglialatela, Celia Ou, Brianna Pagán, Sargent Shriver, Amy Steiker, Michele Thornton, Makhan Virdi, Jessica Nicole Welch, Jess Welch, Erin Robinson, Julia Stewart Lowndes. (2023). NASA-Openscapes/earthdata-cloud-cookbook: NASA EarthData Cloud Cookbook v2023.03.0 (v2023.03.0). Zenodo. https://doi.org/10.5281/zenodo.7786711
```

Comment (stefaniebutland) [reply index.qmd:40]: updated citation to match updated Zenodo author list

**Review by jules32 — APPROVED** [review]
Thank you!

**Comment by stefaniebutland** [comment]
Really strange @jules32 cause my commit does not appear in live Cookbook either (Apr 10, 1pm PT).

Please see and merge this new PR https://github.com/NASA-Openscapes/earthdata-cloud-cookbook/pull/195

**Pull request merged**